Hazel/src/Hazel/Core/Log.cpp

```diff
@@ -1,21 +1,36 @@
 #include "hzpch.h"
-#include "Hazel/Core/Log.h"
+//#include "Hazel/Core/Log.h"		included in hzpch.h already
```

Comment:
This should just be removed, someone can easily find out that it's included in the pch without this here.

Reply:
Cherno mensioned this in one of his videos, we should not rely in Hazel files being included in the precompiled header. The reason we added it to the pch in the first place was because we won't update it often. And especialy in this case where the cpp file should (always) include its header file, so can you please put it back?

Reply:
that is done already .. the include parts of the cpp file are as they have been before my pull request

Reply:
i already put that back yesterday ;)